dangerfile.js

```diff
@@ -1,41 +1,45 @@
 /* global danger, fail, warn, message */
 
 const modifiedFiles = danger.git.modified_files;
-
-// Fail if there isn’t a CHANGELOG entry – should update for every PR
-if (!danger.git.modified_files.includes('CHANGELOG.md')) {
-    const changelogLink = 'https://github.com/justeat/gulp-build-fozzie/blob/master/CHANGELOG.md';
-    fail(`:memo: Please include a CHANGELOG entry. You can find the current version at <a href="${changelogLink}">CHANGELOG.md</a>`);
-}
+const bodyAndTitle = (danger.github.pr.body + danger.github.pr.title).toLowerCase();
+const isTrivial = bodyAndTitle.includes("#trivial");
+
+if (!isTrivial) {
```

Comment:
Is there a way you could exit early here with `if(isTrivial) {...}` rather than nesting the rest of the file?